oss/sdks/ts/qstash/examples/schedules.mdx

```diff
@@ -74,3 +74,36 @@ const client = new Client({ token: "<QSTASH_TOKEN>" });
 const schedules = client.schedules();
 await schedules.delete("scheduleId");
 ```
+
+#### Create a schedule with timeout
+
+Timeout value in seconds to use when calling a schedule URL ([See `Upstash-Timeout` in Create Schedule page](http://localhost:3000/qstash/api/schedules/create)).
+
+```typescript
+import { Client } from "@upstash/qstash";
+
+const client = new Client({ token: "<QSTASH_TOKEN>" });
+const res = await client.publishJSON({
+  url: "https://my-api...",
+  body: { hello: "world" },
+  timeout: 30 // 30 seconds timeout
+});
+```
+
+#### Pause/Resume a schedule
+
+```typescript
+import { Client } from "@upstash/qstash";
+const client = new Client({ token: "<QSTASH_TOKEN>" });
+const scheduleId = "my-schedule"
+
+// pause schedule
+await client.schedules.pause({ schedule: scheduleId });
+
+// check if paused
+const result = await client.schedules.get(createResult.scheduleId);

```

Comment:
we need to use the `scheduleId ` here, `createResult ` does not exist. Same for the resume part